Tests/ExtDataRoot_GridComp.F90

```diff
@@ -38,6 +39,7 @@ MODULE ExtDataUtRoot_GridCompMod
          type(StringStringMap) :: fillDefs
          character(len=ESMF_MAXSTR) :: runMode
          type(timeVar) :: tFunc
+         integer :: delay
```

Comment:
Are you certain you want integer and not real?   Can always change it later, but 1 second is a long time for a computer.

Reply:
Ah I see now that this is in milliseconds.  I'll add  a different comment.